internal/authproxy/serve.go

```diff
@@ -156,6 +154,23 @@ func serve(cmd *cobra.Command, args []string) {
 	}
 	e.Use(middleware.Proxy(middleware.NewRoundRobinBalancer(targets)))
 
-	e.Logger.Info(fmt.Sprintf("Starting proxy for %v", remoteURL))
+	e.Logger.Infof("Starting proxy for remote: %s, cookie key: %s, token of length %d", remoteURL.String(), cookieKey, len(token))
```

Comment:
Shouldn't the port also be part of the printed information ?

Reply:
No because when the server starts it prints the port. There is no need to print it twice.